Mixpanel/MixpanelExceptionHandler.m

```diff
@@ -93,88 +108,66 @@ - (void)addMixpanelInstance:(Mixpanel *)instance {
     [self.mixpanelInstances addObject:instance];
 }
 
-void MPSignalHandler(int signal)
-{
+void MPSignalHandler(int signal, struct __siginfo *info, void *context) {
+    MixpanelExceptionHandler *handler = [MixpanelExceptionHandler sharedHandler];
+
     int32_t exceptionCount = OSAtomicIncrement32(&UncaughtExceptionCount);
-    if (exceptionCount > UncaughtExceptionMaximum)
-    {
-        return;
+    if (exceptionCount <= UncaughtExceptionMaximum) {
+        NSArray *callStack = [MixpanelExceptionHandler backtrace];
+        NSDictionary *userInfo = @{UncaughtExceptionHandlerSignalKey: @(signal),
+                                   UncaughtExceptionHandlerAddressesKey: callStack};
```

Comment:
why do we take the call stack if we dont use it?

Reply:
it's used later https://github.com/mixpanel/mixpanel-iphone/pull/718/files#diff-5d925142f5bb1fb4b058808a4fcdeb10R170

Reply:
right, but we aren't really using it

Reply:
oh sure we can get rid of the logging and all the callstack stuff. i just left it in there because it was there originally